app/src/main/res/drawable/planet_background.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why did you remove some of these assets? I know they're a bit silly now, but I was planning on refactoring the UI to look a bit nicer but haven't gotten around to them. They shouldn't get bundled into the release builds though since they aren't referenced.

Reply:
I basically ran through all the code and resource files trying to eliminate all the warnings and errors. The removed assets had some issues and since they weren't used I made a decision to outright delete them. Seems to have been a bad idea, I'll get them back.